models/bancor/ethereum/bancor_ethereum_trades.sql

```diff
@@ -0,0 +1,169 @@
+{{ config(
+    schema = 'bancor_ethereum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "bancor",
+                                \'["tian7"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2020-01-09' %}
+{% set weth_address = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' %}
+
+WITH conversions AS (
+    SELECT *
+    FROM {{ source('bancornetwork_ethereum', 'BancorNetwork_v6_evt_Conversion') }} t
+    {% if is_incremental() %}
```

Comment:
fyi @augustog & @Hosuke -- we can look at ```project_start_date``` filters to be more important on filtering base tables rather than decoded tables. doesn't hurt to have, doesn't hurt as much without either, since we obtain min(block_date) from the decoded table itself. it's much more efficient on ```transactions``` or ```logs``` as those cover all contracts and can reduce the amount of data